design/decouple-ceph-version.md

```diff
@@ -0,0 +1,158 @@
+# Decoupling the Ceph version
+
+**Targeted for v0.9**
+
+Today the version of Ceph is tied to the version of Rook. Each release of Rook releases a specific version of Ceph that is embedded in the same docker image.
+This needs to be changed such that the version of Ceph is decoupled from the release of Rook. By separating the decision of which version of Ceph will be deployed with Rook, we have a number of advantages:
```

Comment:
It may sound weird, but aren't there also disadvantages?

I don't think they must be added here, though I think it would be good to show the disadvantages in the design so when looking into "better" solutions one can look into removing such (dis-)advantage.

Or can "disadvantages" be seen as covered by the "Coupled (Legacy) Design" section?

Reply:
Yes, I think the disadvantages really are around the complexity, which is in the next section as you mentioned. They are just stated as advantages of the old approach.